src/types.ts

```diff
@@ -0,0 +1,32 @@
+import {GetProjectsQuery, IssueState, IssueStateReason} from "./generated/graphql-operations";
```

Comment:
`graphql-operations` is a pretty big file; just out of curiosity: do we need the whole file?

Reply:
yep, this whole github graphql schema mapped to TypeScript. highly improves support as enables static typing of graphql responses based on default schema + your custom queries. 
For example GetProjectsQuery type is built on top of this query https://github.com/paritytech/roadmap/blob/664732a2830a01ef1283c85bbb89e0c3407e2c75/getProjects.graphql
so when I work with response, it has only those fields that I requested. But yes, we need whole file

we can either ignore this file or keep in git, i preferred to keep